lib/buildsystems/perl.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class PErl < Package
```

Comment:
How about `PERL` all caps?  This mixed case is a bit confusing.

Reply:
Sure.